nft-example/src/lib.rs

```diff
@@ -0,0 +1,232 @@
+#![no_std]
+#![feature(const_btree_new)]
+
+use codec::{Decode, Encode};
+use gstd::{debug, exec, msg, prelude::*, ActorId};
+use primitive_types::{H256, U256};
+use scale_info::TypeInfo;
+
+pub mod payloads;
+pub use payloads::{ApproveForAllInput, ApproveInput, InitConfig, TransferInput};
+
+pub mod state;
+pub use state::{State, StateReply};
+
+use non_fungible_token::base::NonFungibleTokenBase;
```

Comment:
This import is redundant, please remove.

Reply:
It is necessary since ' items from traits can only be used if the trait is in scope'

Reply:
`make pre-commit` errors if any unused imports found.